docs/src/test-parallel-js.md

```diff
@@ -88,6 +88,30 @@ test('runs in parallel 1', async ({ page }) => { /* ... */ });
 test('runs in parallel 2', async ({ page }) => { /* ... */ });
 ```
 
+Alternatively, you can opt-in all tests (or just a few projects) into this fully-parallel mode in the configuration file:
+
+```js js-flavor=js
+// playwright.config.js
+// @ts-check
+
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+const config = {
+  fullyParallel: true,
+};
+
+module.exports = config;
+```
+
+```js js-flavor=ts
+// playwright.config.ts
+import { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  fullyParallel: true,
+};
+export default config;
+```
+
```

Comment:
leaving myself a TODO: to show an example of fullyParallel used for only 1 test file
essentially co-locating an example here so its easier not to miss this awesome feature

## References
- https://playwright.dev/docs/api/class-test#test-describe-configure